roles/iam-user/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Check existance of group policy file.
+  stat:
+    path: "{{playbook_dir}}/opg-ansible/roles/iam-user/files/iam_policy_{{ item }}.json"
```

Comment:
it's created by the jenkins job that runs ansible
The part after roles is a match for the repo structure. 
Only the opg-ansible is variable and needs to match